.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
+python:
+  - "2.7"
+  - "3.3"
```

Comment:
Should 3.3 be listed here (since `scraperwiki-python` is blocking it).

Otherwise, looks good!